<launch>

	<node pkg="camera_self_filter" type="camera_self_filter"  name="camera_self_filter_wide_stereo_left" output="screen">
		<param name="camera_topic" type="string" value="/wide_stereo/right/image_rect_color" />
		<param name="camera_info_topic" type="string" value="/wide_stereo/right/camera_info" />
		<param name="publish_mask" type="bool" value="true" />
		<param name="publish_alpha" type="bool" value="true" />
		<param name="inverted" type="bool" value="false" />
		<param name="robot_description_package_path" type="string" value="$(find pr2_description)" />
		
	</node>

</launch>